src/conversations/index.ts

```diff
@@ -52,4 +53,19 @@ export class Conversations {
 
     return response;
   }
+
+  async evaluateConversation(
```

Comment:
How about simply `evaluate()`? It feels redundant to say conversation twice (conversations.evaluateConversation).